packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.mjs

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable @jessie.js/safe-await-separator */
+import { promises as fs } from 'fs';
+import assert from 'assert';
+
+import {
+  waitForBlock,
+  provisionSmartWallet,
+  getUser,
+  executeOffer,
+} from './upgradeHelpers.mjs';
+import {
+  agd,
+  agoric,
+  agops,
+  agopsLocation,
+  executeCommand,
+} from '../cliHelper.mjs';
+import {
+  HOME,
+  ATOM_DENOM,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+} from '../constants.mjs';
+
+const acceptEC = async () => {
+  console.log('ACTIONS Tickling the wallets so they are revived');
+
+  const govAccounts = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+  let voter = 0;
+  for (const account of govAccounts) {
+    const runs = [1, 2];
+    for (const run of runs) {
+      console.log(`${account}: ${run}: Accepting EC Committee`);
+
+      if (run === 1) {
+        try {
+          const params = ['ec', 'committee', '--send-from', account];
+          await executeCommand(agopsLocation, params, {
+            timeout: 30000,
+          });
+        } catch (error) {
+          console.warn(error.message);
+        }
+      } else {
+        try {
+          await agops.ec('committee', '--send-from', account, '--voter', voter);
```

Comment:
It seems like if we're going to add helpers for agops subcommands, we could turn them into actual javascript calls rather than requiring the user to mimic the CLI version. It seems like the flags could be omitted, so the call here could be `agops.ec(account, voter);`. [I don't know whether 'committee' is a flag or not. If the user has to specify the committee name, for instance, then that could be an extra parameter, and the javascript function's signature would help the caller know what arguments to provide.